extension.html

```diff
@@ -55,6 +55,25 @@ <h2>Introduction</h2>
 	The <code>getCoalescedEvents</code> API (similar to Android <a href="https://developer.android.com/reference/android/view/MotionEvent.html#getHistoricalX(int, int)">getHistorical</a> APIs and iOS <a href="https://developer.xamarin.com/api/member/MonoTouch.UIKit.UIEvent.GetCoalescedTouches/p/MonoTouch.UIKit.UITouch/">GetCoalescedTouches</a> API) exposes all the events that were coalesced into a single event. Coalescing events into a single event enables user agents to prevent processing of the events that don’t get a chance to update the frame content visually. Basically if the javascript handlers process the events and update the screen contents faster than refresh rate only the last update will be rendered and shown in the next frame. However, some applications like drawing apps may want to take advantage of the precise history of events as user moves the pointer on the screen to be able to draw smoother and better looking curves. This API can be used in those scenarios to get all the coalesced events from the event that was dispatched to the javascript.
       </p>
     </section>
+    <section>
+      <h2>Extensions to the Pointer Event types</h2>
+      <p>Below is the new event type that is added to the <a href="https://w3c.github.io/pointerevents/index.html#pointer-event-types">existing pointer event types</a>.</p>
+      <section>
+        <h3>The <dfn><code>pointerrawmove</code> event</dfn></h3>
+        <p>A user agent MUST <a href="https://w3c.github.io/pointerevents/index.html#firing-events-using-the-pointerevent-interface">fire a pointer event</a>
+           named <code>pointerrawmove</code> when a pointing device attribute is changed.
+           As oppose to <code>pointermove</code> which could be fired once every frame and will have coalesced event data,
+           <code>pointerrawmove</code> will get fired as soon as an attribute is changed
+           and as frequent as the javascript can handle the events.
+           Because of that the target of <code>pointerrawmove</code> events might be different from the <code>pointermove</code> events
+           due to the fact that <code>pointermove</code> events are being sent once every frame and the final position of the even
+           which is used for finding the target could be different from its coalesced events.
+           Note that if the javascript handlers are slow, <code>pointerrawmove</code> events can also get coalesced.
```

Comment:
ok, this makes the behavior from specification rather odd. pointermove isn't defined to be only once per frame and then pointerrawmove may get coalesced too, so in practice the events could be exactly the same.

Reply:
The idea was that pointermove will get delayed at the very least until right before rAF. But pointerrawmove will not have that delay at all. However both of these events might get unlimited delay and get coalesced during that time based on the how much the javascript code keeps the code busy. Does this at least make the intention for this event clear and how it is different from pointermove?

Now regarding how to spec it, the problem is that rAF related stuff is not fully specced yet and I believe I need to change HTML spec for this and reference https://html.spec.whatwg.org/#event-loop-processing-model here.

Reply:
I understand what you try to spec, it is just a tad hard to spec when pointermove isn't spec'ed to work exactly how it works in Blink (nor in Gecko). Pointerevents spec says "A user agent MUST fire a pointer event named pointermove when a pointer changes coordinates." That actually kind of prevents firing only at rAF time. So I guess we should fix that too.

Reply:
I tried to explain it better and also added a new pointermove description. Let me know what you think.